data/blog/8-week-cadence.md

```diff
@@ -0,0 +1,86 @@
+---
+title: New Electron Release Cadence
+author: VerteDinde
+date: '2021-07-14'
+---
+
+Beginning in September 2021, Electron will release a new major stable version every 8 weeks.
+
+---
+
+In 2019, Electron moved to a 12 week release cycle to match Chromium's 6 week release cycle. Recently, both Chrome and Microsoft announced changes that made us re-consider Electron's current release cadence:
+
+1. Chromium plans to [release a new milestone every **4 weeks**, starting with Chrome 94 on September 21st, 2021.](https://developer.chrome.com/blog/faster-release-cycle/) This release cadence also adds a new Extended Stable option every 8 weeks, which will contain all updated security fixes.
+
+2. The Microsoft Store will [require Chromium-based apps to be no older than within 2 major versions](https://docs.microsoft.com/en-us/windows/uwp/publish/store-policies#102-security) (for example, if the latest released major version of Chromium is 85, any browser based on Chromium must be on at least Chromium version 83 or higher). This rule includes Electron apps.
+
+**Beginning in September 2021, Electron will release a new major stable version every 8 weeks**, to match Chromium's 8 week Extended Stable releases.
+
+Our first release with Chromium Extended Stable will be **Electron 15** on **September 21st, 2021**.
+
+Knowing the a release cadence change will impact other downstream applications, we wanted to let our developer community know as soon as possible. Read on for more details about our 2021 release schedule.
+
+## Moving to Extended Stable
+
+Given that our original Electron 15 release targeted a non-Extended Stable version (Chromium's Extended Stable versions are based on their even-numbered versions), we needed to change our original target release date. However, an Electron app must use the most recent 2 major versions of Chromium to be accepted to the Microsoft Store. With these two requirements, our Releases WG faced a timing dilemma. 
+
+Below is the planned release schedule for 2021:
```

Comment:
The wording here doesn't differentiate itself very much from the "current release schedule" section below. Someone skimming the tables might get confused.

Reply:
This is a good point. Maybe we can combine the tables or use one of them 🤔 

Suggestion:
* Delete first table from ##Moving to Extended Stable section. Replace with something like "E15 will target a single chromium bump to M94."
* Keep second table in ## 2021 Plan for Releases as is. It's more descriptive and has almost the same info as the first table.

Reply:
Done! Removed the table and section, kept the note about why we needed to do the single Chromium bump.